fuzz/generate-files.sh

```diff
@@ -102,7 +102,7 @@ $(for name in $(listTargetNames); do echo "          $name,"; done)
     runs-on: ubuntu-24.04
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/download-artifact@v4
+      - uses: actions/download-artifact@v5
```

Comment:
This is an unrelated change mate. Can it go as a separate patch up front? In case you wonder why I say this, its because to review I had to go digging as to _why_ this change was being done and whether it was correct, all these things took time and brain power that would not have been required if it was a separate patch with a commit log explaining it. Thanks